### ISSUE TITLE
Expanded dominant children?

### ISSUE THREAD

**Issue by doppio** [PixelToast/flutter-boxy#4]
Does Boxy support Dominant children that are also Expanded? I have a row where one child should expand to fill as much horizontal space as possible, and the other children should match its height.

**Comment by PixelToast** [comment]
Yes! Use BoxyFlexible:

```dart
BoxyFlexible(
  fit: FlexFit.tight,
  dominant: true,
  child: ...,
)
```

**Comment by PixelToast** [comment]
Keeping open since documentation can be improved here

**Comment by PixelToast** [comment]
Fixed in 317783f7e064ae4aadd39715f6770502403f3908

As of the next release you can now use the `Dominant.expanded` constructor:

```dart
Dominant.expanded(
  child: ...,
)
```
